### PR TITLE
gives nicotine free cigarettes a expanded description to make more sense

### PR DESCRIPTION
**Pull request by boodaliboo**
"A Tobacco-free cigarette, it contains Chamomile, Lemon Balm, and Catnip."
this is the new description as a cigarette with no nicotine and tobacco is impossible
:cl:  
tweak: tweaked nicotine free description
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
tell me if i did this wrong

**Comment by Marmio64** [comment]
contains chamomile, lemonbalm, and catnip, yet still only contains carbon reagents?  Maybe just keep it ambiguous or say it simply contains nicotine free tobacco.

**Comment by boodaliboo** [comment]
> contains chamomile, lemonbalm, and catnip, yet still only contains carbon reagents? Maybe just keep it ambiguous or say it simply contains nicotine free tobacco.

nicotine free tobacco doesnt exist though, and there isnt really any reagents that would reflect the herbal taste

**Comment by ynot01** [comment]
uncapitalize the names they arent proper nouns please

**Comment by Marmio64** [comment]
> > contains chamomile, lemonbalm, and catnip, yet still only contains carbon reagents? Maybe just keep it ambiguous or say it simply contains nicotine free tobacco.
> 
> nicotine free tobacco doesnt exist though, and there isnt really any reagents that would reflect the herbal taste

nicotine free tobacco doesn't exist cause no ones added it yet, bruh. And it's not like it has to be an actual plant...
And, you can just like, you know, make a reagent that will reflect it.

File: code/game/objects/items/cigs_lighters.dm [hunk 1]
```diff
@@ -333,7 +333,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	list_reagents = list(/datum/reagent/drug/nicotine = 20, /datum/reagent/medicine/regen_jelly = 15, /datum/reagent/drug/krokodil = 4)
 
 /obj/item/clothing/mask/cigarette/nonico
-	desc = "A nicotine-free cigarette."
+	desc = "A Tobacco-free cigarette, it contains Chamomile, Lemon Balm, and Catnip."
```
Comment (ToasterBiome) [inline code/game/objects/items/cigs_lighters.dm]:
```suggestion
	desc = "A tobacco-free cigarette, it contains chamomile, lemon balm, and catnip."
```

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

File: code/game/objects/items/cigs_lighters.dm (hunk 1 above)
Comment (ToasterBiome) [reply code/game/objects/items/cigs_lighters.dm]: i think the "ingredients" are fine, they literally don't have to mean anything. just don't capitalize them

**Comment by boodaliboo** [comment]
> > > contains chamomile, lemonbalm, and catnip, yet still only contains carbon reagents? Maybe just keep it ambiguous or say it simply contains nicotine free tobacco.
> > 
> > 
> > nicotine free tobacco doesnt exist though, and there isnt really any reagents that would reflect the herbal taste
> 
> nicotine free tobacco doesn't exist cause no ones added it yet, bruh. And it's not like it has to be an actual plant...
> And, you can just like, you know, make a reagent that will reflect it.

im not making a reagent for this its fine as is it was just made to be a description change as those are common ingredients in herbal smoke mixes

**Comment by Hopekz** [comment]
> nicotine free tobacco doesnt exist though, 

Imo it's funny to think that the company is just lying to their audience and telling them they have nicotine free tobacco just like companies in China that sell diet water

File: code/game/objects/items/cigs_lighters.dm [hunk 2]
```diff
@@ -931,4 +931,4 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 /obj/item/clothing/mask/cigarette/lit/Initialize()
     . = ..()
     light() // These cigarettes start lit.
-    
\ No newline at end of file
+
```
Comment (JamieD1) [inline code/game/objects/items/cigs_lighters.dm]:
```suggestion
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**